src/ar_meta_db.erl

```diff
@@ -1,91 +1,179 @@
+%% @doc Defines a small in-memory metadata table for Arweave nodes.
+%% Typically used to store small peices of globally useful information
+%% (for example: the port number used by the node).
+%% @end
+
 -module(ar_meta_db).
--export([start/0, get/1, put/2, keys/0, remove_old/1,remove_old/2, increase/2]).
+-behaviour(gen_server).
+-define(SERVER, ?MODULE).
```

Comment:
Why did you define the `SERVER` constant? You mostly use `?MODULE` anyways.

Reply:
I don't have a good reference. I think it something I have picked up in a book. I often use it for for singletons. 

```gen_server:start_link({local, ?SERVER}, ?MODULE, [], []).```

Reply:
I think it is a common pattern. For example here:
https://github.com/stevenchan/minipush/blob/478046fc0962a13b6cb93aa8d6f6909e811271d4/src/supervisor/minipush_sup.erl

Reply:
I found a reference. *Erlang and OTP in Action* by Martin Logan pg 103.
>>>
"[Sets SERVER to module name] you may want to change that at some point, so you shouldn't assume that the server name will always remain the same as the module name"